bindings_node/src/conversations.rs

```diff
@@ -219,10 +237,18 @@ impl Conversations {
   }
 
   #[napi(js_name = "createDm")]
-  pub async fn find_or_create_dm(&self, account_address: String) -> Result<Conversation> {
+  pub async fn find_or_create_dm(
+    &self,
+    account_address: String,
+    options: Option<CreateDMOptions>,
+  ) -> Result<Conversation> {
+    let metadata_options = options.unwrap_or(CreateDMOptions {
```

Comment:
Nit: could just derive default on createdmoptions and do `.unwrap_or_default()` here.